### DISCUSSION TITLE
Incorrect recording directories timestamp (name)

### DISCUSSION THREAD

**Discussion by ABATAPA** [blakeblackshear/frigate#6345] [General]
Hello all!
I see that in the /recordings folder, the directories corresponding to the hour are created according to UTC time (without offset).
Current:
drwxr-xr-x 3 root root 4096 May  2 12:00 09

'09' corresponds to the UTC hour (timezone = UTC+3), but the date of the change is correct. And the name of temporary files in the cache directory is created correctly:
-rw-r--r-- 1 root root 8388656 May  2 12:33 Zodikam-20230502123332.mp4
This matches the pattern passed by ffmpeg:
/tmp/cache/Zodikam-%Y%m%d%H%M%S.mp4

After writing to a temporary file, it is transferred to
/storage/recordings/2023-05-02/**09**/Zodikam/33.32.mp4, but not .../12/...

Configured:
volumes:
      - /etc/localtime:/etc/localtime:ro
      
The time format and the time itself in the container are correct.
In the interface, the time is correct, the time of the recordings is displayed correctly.
ffmpeg creates temporary files with the correct name. Frigate then moves it to the wrong directory .

What could be the problem?
It's as if Frigate doesn't take localtime  when creating a directory.

**Comment by blakeblackshear** [comment]
From the release notes:

> Recording segments are now stored in UTC to avoid issues with DST

**Reply by ABATAPA** [reply]
Is it possible to make it configurable via a parameter?

**Reply by blakeblackshear** [reply]
It's not. The problem is that daylight savings time changes cause directories to be reused and recordings to be overwritten during time changes.

Can you explain why it matters what frigate's internal database structure looks like? These files aren't meant to be interacted with directly. They are binary blob storage for the database.
